cvmfs/cvmfs_config

```diff
@@ -245,6 +312,12 @@ cvmfs_chksetup() {
     fi
   done
 
+  # Check that CVMFS_USER is set
+  if [ x"$CVMFS_USER" = x"" ]; then
```

Comment:
Not sure. The root cause for this addition is that `$CVMFS_USER` was used before it was checked for existence. Personally I find this check more understandable than checking for `default.conf` where I have no detailed idea what is inside there.